README.md

```diff
@@ -96,4 +104,43 @@ They are Headers that apply to all requests without having to manually include t
 
 * `Http.SetSuperHeader(string key, string value)`
 * `Http.RemoveSuperHeader(string key)` returns `bool`
-* `Http.GetSuperHeaders()` returns `Dictionary<string, string>`
\ No newline at end of file
+* `Http.GetSuperHeaders()` returns `Dictionary<string, string>`
+
+
+## JSON response Example
+In this given example, the `response.Text` from `http://mywebapi.com/user.json` is:
+```json
+{
+    "id": "92",
+    "username": "jason"
+}
+```
+
+Create a serializable class that maps the data from the json response to fields
+```c#
+[Serializable]
+public class User
+{
+    [SerializeField]
```

Comment:
I do get this is an example but aren't public fields always serialized and therefore [SerializeField] is redundant?

Reply:
They are but I think its best to be explicit with it.